lib/types.ts

```diff
@@ -39,6 +39,7 @@ export interface Highlights {
   pr_collaborated: number;
   issue_assigned: number;
   issue_opened: number;
+  newContributor?: boolean;
```

Comment:
@rhithesh let's store the `first_activity` date instead of a boolean field as that'd allow us to show "Contributing since <date>" in contributors' profiles, and filtering can be done by doing a comparison of is within the last N days or not against this field.